components/tools/OmeroJava/test/integration/LightAdminRolesTest.java

```diff
@@ -2530,11 +2530,11 @@ public void testResetEnumerationsbyRestrictedSystemUser() throws Exception {
         }
         //not all enum are used so we should have deleted at least one
         Assert.assertTrue(count > 0);
-        types = types_svc.allEnumerations(Detector.class.getName());
+        types = types_svc.allEnumerations(DetectorType.class.getName());
         Assert.assertEquals(types.size(), (n-count));
         //reset the deleted enumerations
-        types_svc.resetEnumerations(Detector.class.getName());
-        types = types_svc.allEnumerations(Detector.class.getName());
+        types_svc.resetEnumerations(DetectorType.class.getName());
+        types = types_svc.allEnumerations(DetectorType.class.getName());
         //We should be back to at the original list. Other enum might have been
```

Comment:
``We should be back to at the original list`` -> either ``at`` or ``to`` - I think both ``at`` and ``to`` is not correct - @mtbc ?

Reply:
Indeed.